.github/workflows/run-e2e-tests.yml

```diff
@@ -3,6 +3,12 @@ on:
   workflow_dispatch: {}
 jobs:
   run:
+    #########################
+    # Force Github action to run only a single job at a time (based on the group name)
+    # This is to prevent "race-condition" in building e2e tests infrastructure
+    #########################
+    concurrency:
+      group: e2e-tests
```

Comment:
For that one I would prefer we fix the root cause preventing them to run in parrallel : #497

Reply:
Will revert that in a dedicated PR fixing #497